routes/test.js

```diff
@@ -0,0 +1,10 @@
+const crypto = require("crypto");
+
+const secret = "abcdefg";
+const hash = crypto
+  .createHmac("sha256", secret)
+  .update("I love cupcakes")
+  .digest("hex");
+console.log(hash);
+// Prints:
+//   c0fa1bc00531bd78ef38c628449c5102aeabd49b5dc3a2a516ea6ea959d6658e
```

Comment:
Why create this file if we are not using it anywhere?
Use `bcrypt` for hashing passwords.
https://www.npmjs.com/package/bcrypt